package.json

```diff
@@ -37,7 +38,9 @@
             "packages/ethereum-contracts",
             "packages/js-sdk",
             "packages/sdk-core",
-            "packages/subgraph"
+            "packages/sdk-redux",
+            "packages/subgraph",
+            "examples/sdk-redux-react-typecript"
```

Comment:
sdk-redux-react-typescript should not be included here, since it's an example project, in order to be consistency with other example projects.

(also maybe let's consider to rename it to a better project name, maybe `example/sdk-redux-demo/packages.json` ?)

To the practical concern about how to develop this example project going forward:

If `sdk-redux` or `sdk-core` changes are required:

A. if the changes are small, consider to use the normal process to publish a new canary package of the SDK
B. if the changes are big enough that need a bit more testing together with the example project itself
  1. Create a feature branch
  2. Change locally `pacakge.json` and `example/sdk-redux-demo/packages.json` to make it temporarily workspace package during the development process
  3. After development and testing, extract the sdk changes to a new feature branch, and use the normal process to make sure that new sdk changes are clear to everyone
  4. After the canary builds (or even a release build) of the sdk is available, revert what's done in (2), and make the example project work again as independent project.

This way we can force us to have the chance of understanding what's the impact of any sdk changes to the end users.

Reply:
I'm OK with it if I can do a manual `yarn link @superfluid-finance/sdk-redux` once to the example project and then change branches and the link sticks.